bulbs/content/management/commands/migrate_to_ia.py

```diff
@@ -0,0 +1,31 @@
+from django.core.management.base import BaseCommand
+
+from bulbs.content.models import Content, FeatureType
+
+import timezone
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('feature', nargs="+", type=str)
+
+    def handle(self, *args, **options):
+        feature_types = FeatureType.objects.all()
+
+        feature = options['feature'][0]
+        if feature:
+            feature_types.objects.filter(slug=feature)
+
+        for ft in feature_types:
+            if ft.instant_article:
+
+                # All published content belonging to feature type
+                content = Content.objects.filter(
+                    feature_type=ft,
+                    published__isnull=False,
+                    published__lte=timezone.now())
+
+                for c in content:
+                    # save to trigger celery task to post to Instant Article API
+                    c.save()
```

Comment:
actually, i think just calling the celery task will solve the issue, since it gets the content as the first thing in the task. if the content is updated after that, `save()` will be called, which should post the article to the API again.